s4e-web/src/app/views/map-view/state/product/product.service.ts

```diff
@@ -110,7 +109,13 @@ export class ProductService {
 
     this.http.get<string[]>(`${this.CONFIG.apiPrefixV1}/products/${activeProduct.id}/scenes/available`,
       {params: {tz: this.CONFIG.timezone, yearMonth: dateF}})
-      .pipe(finalize(() => this.store.setLoading(false)))
+      .pipe(
+        finalize(() => this.store.setLoading(false)),
+        catchError(error => {
+          this.store.setError(error);
+          return throwError(error);
+        }),
```

Comment:
As I mentioned before, this piece of code repeats very often, so it should be extracted into separate rxjs operator